components/form/form-error-summary.js

```diff
@@ -19,10 +21,12 @@ class FormErrorSummary extends LocalizeCoreElement(LitElement) {
 			:host {
 				display: block;
 			}
-
 			:host([hidden]) {
 				display: none;
 			}
+			:host([_has-bottom-margin][_has-errors]) {
+				margin-block-end: 1rem;
```

Comment:
I think this component should have always had bottom margins since its inception, but consumers have likely worked around this so it's likely too late to change this.